vault/keysign.go

```diff
@@ -214,34 +226,70 @@ func (t *DKLSTssService) keysign(sessionID string,
 			t.logger.Error("failed to free keyshare", "error", err)
 		}
 	}()
-	md5Hash := md5.Sum([]byte(message))
-	messageID := hex.EncodeToString(md5Hash[:])
 	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
 	defer cancel()
-	// retrieve the setup Message
-	encryptedEncodedSetupMsg, err := relayClient.WaitForSetupMessage(ctx, sessionID, messageID)
-	if err != nil {
-		return nil, fmt.Errorf("failed to get setup message: %w", err)
+
+	var encryptedEncodedSetupMsg string
```

Comment:
would rather to keep the unencryped setupMsg , so for initiate device , it doesn't need to the unnecessary decryption

Reply:
that's fine ,